model/project_ref.go

```diff
@@ -172,6 +172,8 @@ const (
 	ProjectTriggerLevelBuild = "build"
 )
 
+var ErrProjectNotFound = errors.New("error finding the project")
```

Comment:
should probably call this ProjectRef

Reply:
Do you mean call it ErrProjectRefNotFound?

Reply:
These errors are user-facing, so we should not call it project ref (I would word it something like "unable to find project _____")